kenmerken-nummeraanduiding-in-onderzoek.feature

```diff
@@ -0,0 +1,63 @@
+# language: nl
+Functionaliteit: indicatie één of meerdere kenmerken van nummeraanduiding in onderzoek
+Design decisions:
+
+1. indicatie inOnderzoek (ook individuele) niet in response opnmen als de Resource of één of meerdere kenmerken van de Resource niet in onderzoek zijn

```

Comment:
Bij BAg 2.0 kunnen alleen nog individuele attributen van objecten in onderzoek zijn. Er stat nu dat ook een resource in onderzoek kan zijn maar is niet correct. Bedoel je hier dat als er geen onderzoek naar een attribuut is, hier geen InOnderzoek indicatie van opgenomen moet worden in de response? Dat lijkt mij idd. een goed beginsel. Als dit inhoud dat er helemaal geen onderzoeken naar attributen van een object zijn, dan moet er dus helemaal geen inonderzoek informatie (indicaties) worden geretourneerd? Lijkt mij prima. 
De beschrijving zou nu ook geïnterpreteerd kunnen worden: als er één attribuut in onderzoek is, wordt er helemaal geen inonderzoek informatie geretourneerd, maar dat moeten we niet doen. Anders gezegd: als er één of meerdere attributen in onderzoek is/zijn, dan moet alle informatie geretourneerd worden.

Reply:
Wat wordt bedoeld is om het niet in onderzoek zijn, niet expliciet op te nemen in de response.